docs/maintenance.rst

```diff
@@ -17,6 +17,29 @@ Assuming your new coefficient file has the same format, the process is simple:
 3. Install the python package from the command line
    (see :ref:`installation-cmd`)
 
+Updating ``apexsh.dat``
+-----------------------
+
+After updating the IGRF coefficients file the ``apexsh.dat`` file needs to be
+rebuilt. This file is what makes apexpy so perfromant. For more details, see
+Emmert et al. [2010] [1]_.
+
+Updating ``apexsh.dat`` is done by modifying and building the fortran source code
+in the ``apexpy/src/fortranapex`` directory. Working in that directory:
+
+1. Make sure a copy of the latest IGRF coefficient file is present in the
+   directory.
```

Comment:
```suggestion
1. Make sure a copy of the latest IGRF coefficient file is present in the
   selfsame directory.
```

Thought it might be helpful to be very clear that a copy is needed in the apexpy/src/fortranapex directory.